tests/syntax-tests/highlighted/Rust/output.rs

```diff
@@ -93,11 +93,14 @@
 [38;2;248;248;242m                            [0m[3;38;2;166;226;46mNone[0m[38;2;248;248;242m [0m[38;2;249;38;114m=>[0m[38;2;248;248;242m [0m[38;2;248;248;242m{[0m
 [38;2;248;248;242m                                p[0m[38;2;248;248;242m.[0m[38;2;102;217;239marg[0m[38;2;248;248;242m([0m[38;2;230;219;116m"[0m[38;2;230;219;116m--no-init[0m[38;2;230;219;116m"[0m[38;2;248;248;242m)[0m[38;2;248;248;242m;[0m
 [38;2;248;248;242m                            [0m[38;2;248;248;242m}[0m
-[38;2;248;248;242m                            [0m[3;38;2;166;226;46mSome[0m[38;2;248;248;242m([0m[38;2;248;248;242mversion[0m[38;2;248;248;242m)[0m
```

Comment:
IMHO it is better to leave the test files (both source and highlighted) unchanged, because that is the safest way to ensure there are no regressions. Plus, it is unnecessary effort to keep the test source file in sync with the "real" source file. The only purpose of the test file is to make sure highlight keeps working.